src/MessagePack.Experimental/PrimitiveArrayResolver.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using MessagePack.Formatters;
+
+namespace MessagePack.Experimental.Resolvers
```

Comment:
If we use the actual namespace and type names that we would in the primary library, we can eventually *move* these types from Experimental to the main library without a binary breaking change in the future. Any concerns with dropping `Experimental` from the namespace?

Reply:
I do not have any concerns.
I had forgotten the perspective "binary breaking change in the future".
Thank you for your dropping `Experimental`!